src/material/slider/slider.md

```diff
@@ -17,8 +17,10 @@ respectively. The initial value is set to the minimum value unless otherwise spe
 
 ### Selecting a range
 A `<mat-slider>` can be converted into a range slider by projecting both a `matStartThumb` and a
-`matEndThumb` into it. Each of the thumbs has an independent value, but they won't be allowed to
-overlap and they're still bound by the same `min` and `max` from the slider.
+`matEndThumb` into it. Each of the thumbs has its own value, but both are still
+constrained by the slider's `min` and `max` values. `matStartThumb` cannot have a value
```

Comment:
> `matStartThumb` cannot

Should be: "The `matSliderStartThumb` cannot". Same for other references to the start & end thumb directives

Reply:
fixed

Reply:
I think we're still missing the "The" here and before other references to the other thumb directives

Reply:
Sorry about that! Should be there now.